CHANGELOG.md

```diff
@@ -5,17 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.47.0] - 2025-10-16
 
 ### Changed
-- Support OpenCTI version 6.8.3 - No graphql changes
+- Support OpenCTI version 6.8.6
```

Comment:
Why is there a gap between `6.8.3` and `6.8.6` ?

Reply:
The update job was blocked because of a mismatch between go version. I'll do those ones manually.